src/iolStateMachineHandler/src/module.cpp

```diff
@@ -2013,6 +2013,9 @@ void Manager::updateMemory()
         for (map<string,Tracker>::iterator it=trackersPool.begin(); it!=trackersPool.end(); it++)
             it->second.prepare();
         
+        // Classification scores for each object
+        std::map<string, double> scoresMap;
```

Comment:
What do you need `scoresMap` for?
Wouldn't it be easier to deal with just `score`?

Reply:
I need this because the computation of the scores and the writing operation in OPC are made in two separate for loops.
And these two loops do not necessarily have the same ordering.
First loop (score computation and object name identification) is ordered by blob names (blob_0, blob_1, ...) while the second loop (writing to OPC) is ordered according to the map<string, Tracker> trackersPool, which I think is ordered alphabetically by object names.